### DISCUSSION TITLE
IMS Registered in Perú with carrier Claro

### DISCUSSION THREAD

**Discussion by NathanHayball** [kyujin-cho/pixel-volte-patch#93] [Carrier-specific Experiences]
I was able to get it working on South America - Perú with the carrier Claro on the Pixel 7. 

I tested it on Movistar but it didn't register, although I did notice that LTE+ worked in a couple of areas (in Lima) but I wouldn't call it reliable.

Claro: 
![Screenshot_20230325-083246~2](https://user-images.githubusercontent.com/128911586/227721668-91f29a15-1e3b-4147-9910-dbf1b6c93eed.png)

Movistar: 
![Screenshot_20230324-131252~4](https://user-images.githubusercontent.com/128911586/227721919-438a2ab8-beea-4d9d-b0b5-8d52fe24ae0c.png)

@kyujin-cho

**Comment by NathanHayball** [comment]
Update: LTE+ started working on Claro. Went to a deserted area with no signal and and when I got back home LTE+ was working, so maybe reseting the network did it.

**Comment by ramzi** [comment]
Is VoWiFi working for you with Claro? I haven't been able to make it work. VoWiFi says unavailable.

**Reply by NathanHayball** [reply]
No luck for me either

**Reply by msilcher** [reply]
> Is VoWiFi working for you with Claro? I haven't been able to make it work. VoWiFi says unavailable.

I made it work with Claro in Argentina, had to call support and they pushed an update on their side. After rebooting the phone it worked. It might be worth a try...

P.S: You need to create the "ims" APN

**Comment by kyujin-cho** [comment]
Thanks for the report! I've updated the compat matrix.
